### PR TITLE
Remove command execution from the extended image

### PR DESCRIPTION
**Pull request by mkumatag**
Fixes: https://github.com/openshift/origin/issues/24068

### CODE REVIEW

File: test/extended/images/extract.go [hunk 1]
```diff
@@ -76,7 +76,7 @@ var _ = g.Describe("[Feature:ImageExtract] Image extract", func() {
 			mkdir -p /tmp/test
 			oc image extract --insecure %[2]s/%[1]s/1:busybox --path=/:/tmp/test
 			[ -d /tmp/test/etc ] && [ -d /tmp/test/bin ]
-			[ -f /tmp/test/bin/ls ] && /tmp/test/bin/ls /tmp/test
+			[ -f /tmp/test/bin/ls ]
```
Comment (bparees) [inline test/extended/images/extract.go]: hmm... is there something else we can do to confirm the content was extracted successfully?  perhaps extract a text file and confirm the content?  i'd prefer to have some assurance we are not corrupting or truncating the content on extraction.

Comment (mkumatag) [reply test/extended/images/extract.go]: I'm not really sure how to that! here we are just importing files from baseimage and checking the content! I'm wondering is there a way where we bundle something in the baseimage and extract and check!

Comment (mkumatag) [reply test/extended/images/extract.go]: How about grepping `root` in the `/etc/passwd` file?

Comment (bparees) [reply test/extended/images/extract.go]: i was hoping there might be something already in the busybox image you could check, but i guess there's not much in there.  can you at least confirm the extracted file is non-zero sized?

Comment (bparees) [reply test/extended/images/extract.go]: > How about grepping root in the /etc/passwd file?

sounds good to me.

Comment (mkumatag) [reply test/extended/images/extract.go]: Pushed the changes, hope it works fine 🤞

**Pull request merged**